scripts/latest-news/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "fast-xml-parser": "^4.3.2"
+  }
+}
```

Comment:
I doubt that we need a separate package.json. I recall we tried to use a separate package file and removed it.
But on the other hand, it speeds up the build time. @krutilov WDYT?